physical/raft/raft_autopilot.go

```diff
@@ -9,6 +9,7 @@ import (
 	"os"
 	"strconv"
 	"sync"
+	"sync/atomic"
```

Comment:
I suggest using uber's atomic instead, so you don't have to worry about alignment.

Reply:
Done.